app/authentification/utils.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.shortcuts import redirect
+from django.contrib.auth import login
+from django.utils.encoding import force_text
+from django.utils.http import urlsafe_base64_decode
+from .tokens import account_activation_token
+from django.contrib.auth.models import User
```

Comment:
`from django.contrib.auth import get_user_model` is a better way to refer to User. The reason is:
* https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#referencing-the-user-model

Reply:
same thing as app/authentification/forms.py